languages/csharp/exercises/concept/numbers-signedness/.docs/after.md

```diff
@@ -0,0 +1,13 @@
+# After
+
+Integral numeric types have signed and unsigned variants. Unsigned variants can represent double the range of their signed counter part. These days for most programs memory usage is not a problem so choosing bigger signed type is in most cases a better choice. When reaching the limits of the `long` type choosing to use the `ulong` type can still be a good choice.
```

Comment:
```suggestion
Integral numeric types have signed and unsigned variants. Unsigned variants can represent double the range of their signed counter part. These days for most programs memory usage is not a problem so choosing bigger signed type is in most cases a better choice. When reaching the limits of the `long` type choosing to use the `ulong` type can still be a good choice.
```

I'm not sure about how this sentence is going to be interpreted by students... I'd advise using `int` until you know better.